src/Apps/Home/Components/HomeAuctionLotsForYouRail.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  ShelfArtworkFragmentContainer,
+  ShelfArtworkPlaceholder,
+} from "Components/Artwork/ShelfArtwork"
+import {
+  ActionType,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+
+import { HomeAuctionLotsForYouRail_artworksForUser$data } from "__generated__/HomeAuctionLotsForYouRail_artworksForUser.graphql"
+import { HomeAuctionLotsForYouRailQuery } from "__generated__/HomeAuctionLotsForYouRailQuery.graphql"
+import { extractNodes } from "Utils/extractNodes"
+
+interface HomeAuctionLotsForYouRailProps {
+  artworksForUser: HomeAuctionLotsForYouRail_artworksForUser$data
+}
+
+const HomeAuctionLotsForYouRail: React.FC<HomeAuctionLotsForYouRailProps> = ({
+  artworksForUser,
+}) => {
+  const { trackEvent } = useTracking()
+
+  const artworks = extractNodes(artworksForUser)
+
+  if (!artworks || artworks?.length === 0) {
+    return null
+  }
+
+  return (
+    <Shelf>
+      {artworks.map((artwork, index) => {
+        if (!artwork) {
+          return <></>
+        }
+
+        return (
+          <ShelfArtworkFragmentContainer
+            artwork={artwork}
+            key={index}
+            contextModule={ContextModule.auctionLots}
+            lazyLoad
+            onClick={() => {
+              //TODO: add tracking for Auction Lots For You
+              const trackingEvent: ClickedArtworkGroup = {
+                action: ActionType.clickedArtworkGroup,
+                context_module: ContextModule.auctionLots,
+                context_page_owner_type: OwnerType.home,
+                destination_page_owner_id: artwork.internalID,
+                destination_page_owner_slug: artwork.slug,
+                destination_page_owner_type: OwnerType.artwork,
+                type: "thumbnail",
+              }
+              trackEvent(trackingEvent)
+            }}
+          />
+        )
+      })}
+    </Shelf>
+  )
+}
+
+const PLACEHOLDER = (
+  <Skeleton>
+    <Shelf>
+      {[...new Array(8)].map((_, i) => {
+        return <ShelfArtworkPlaceholder key={i} index={i} />
+      })}
+    </Shelf>
+  </Skeleton>
+)
+
+export const HomeAuctionLotsForYouRailFragmentContainer = createFragmentContainer(
+  HomeAuctionLotsForYouRail,
+  {
+    artworksForUser: graphql`
+      fragment HomeAuctionLotsForYouRail_artworksForUser on ArtworkConnection {
+        edges {
+          node {
+            internalID
+            slug
+            ...ShelfArtwork_artwork
+          }
+        }
+      }
+    `,
+  }
+)
+
+export const HomeAuctionLotsForYouRailQueryRenderer: React.FC = () => {
+  const { relayEnvironment } = useSystemContext()
+
+  return (
+    <SystemQueryRenderer<HomeAuctionLotsForYouRailQuery>
+      lazyLoad
+      environment={relayEnvironment}
+      query={graphql`
+        query HomeAuctionLotsForYouRailQuery {
+          artworksForUser(
+            includeBackfill: true
+            first: 20
+            maxWorksPerArtist: 3
```

Comment:
These arguments - `maxWorksPerArtist` and `version` can be left out for the auction-lot recs (`maxWorksPerArtist` is unused and `version` will default to the current version for these recs, which is actually "A" - https://github.com/artsy/vortex/blob/6a74d2050d2a9ab752dd77ecbd751eb53cd6c115/app/models/auction_lot_for_you_recommendation.rb#L2)